File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int prev = nums[0];
        for (int i=1;i<nums.size();i++) {
            nums[i] = nums[i] + prev;
            prev = nums[i];
        }
        return nums;
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int n = nums.length;
        int[] runSum = new int[n];
        runSum[0] = nums[0];
        for(int i=1;i<n;i++){
            runSum[i] = nums[i]+runSum[i-1];

        }
        return runSum;
    }
}